docs/source/user-guide/concepts/packages.rst

```diff
@@ -29,6 +29,35 @@ package. Directories are not included. Directories are created
 and removed as needed, but you cannot create an empty directory
 from the tar archive directly.
 
+.conda file format
+==================
+
+
+The .conda file format was introduced in conda 4.7 as a more
+compact, and thus faster, alternative to a tarball. 
+
+The .conda file format consists of an outer, uncompressed
+ZIP-format container, with two inner compressed .tar files.
+The inner files are compressed using Zstandard (ztsd). 
```

Comment:
Paragraph suggestions:

For the .conda format's initial internal compression format support, 
we chose Zstandard (zstd).  The actual compression format used does not
matter, as long as the format is supported by libarchive.  The compression
format may change in the future as more advanced compression algorithms are
developed, and no change to the .conda format is necessary.  Only an updated 
libarchive would be required to add a new compression format to .conda files.